config/family_config.csv

```diff
@@ -1,26 +1,31 @@
-Inheritor,Type,Asaba_Rank
-son,A,1
-son_of_son,A,2
-father,FA,3
-father_of_father,FA,4
-brother,FA,5
-paternal_halfbrother,A,6
-maternal_halfbrother,F,
-nephew_brother,A,7
-nephew_paternal_halfbrother,A,8
-fathers_brother,A,9
-fathers_paternal_halfbrother,A,10
-son_of_fathers_brother,A,11
-son_of_fathers_paternal_halfbrother,A,12
-husband,F,
-atiq,A,13
-daughter,FA,
-daughter_of_son,FA,
-mother,F,
-grandmother_father,F,
-grandmother_mother,F,
-sister,FA,
-paternal_halfsister,FA,
-maternal_halfsister,F,
-wife,F,
-mutiqa,A,13
\ No newline at end of file
+Inheritor,Type,Asaba_Rank,descendant,blocked_by
+son,A,1,M,
+son_of_son,A,2,M,

```

Comment:
just so I fully understand. In this case are we saying `son_of_son` is not blocked by anyone or this naturally assumes blocked by rank above which is `son`?

Reply:
The blocked_by column is totally empty, it's yet to be filled and isn't used yet in the code. It's a placeholder for future work, whoever wants to fill the F's can fill them, that'll be a ticket. A's don't need for now, they're blocked based on rank.